src/main/java/com/amazon/ion/impl/macro/Macro.kt

```diff
@@ -54,11 +54,11 @@ sealed interface Macro {
         }
     }
 
-    enum class ParameterCardinality(@JvmField val sigil: Char) {
-        ZeroOrOne('?'),
-        ExactlyOne('!'),
-        OneOrMore('+'),
-        ZeroOrMore('*');
+    enum class ParameterCardinality(@JvmField val sigil: Char, @JvmField val canBeVoid: Boolean, @JvmField val canBeMulti: Boolean) {
```

Comment:
🗺️ Added these because as I was working on the macro evaluator, I came across situations where I wanted to simply check "can this be void" or "can this be multiple expressions".